src/runtime/auto-prompt-manager.ts

```diff
@@ -997,7 +1023,16 @@ export class AutoPromptManager {
    */
   private async getActionImpressions_(actionType: string): Promise<number[]> {
     if (!ACTION_TO_IMPRESSION_STORAGE_KEY_MAP.has(actionType)) {
-      // TODO: handle missing storage key for action
+      this.eventManager_.logEvent({
+        eventType:
+          AnalyticsEvent.EVENT_ACTION_IMPRESSIONS_STORAGE_KEY_NOT_FOUND_ERROR,
+        eventOriginator: EventOriginator.SWG_CLIENT,
+        isFromUserAction: false,
+        additionalParameters: {
+          publicationid: this.pageConfig_.getPublicationId(),
```

Comment:
Isn't publication_id already logged by default?
http://google3/logs/proto/subscribewithgoogle/client/subscribewithgoogle_client_extension.proto;l=47;rcl=568642159

Reply:
I see, so I can exclude this?

Reply:
Yes, it seems unnecessary. anyway it won't be logged.